docs/INSTALL.md

```diff
@@ -100,6 +99,18 @@ sudo apt-get update && sudo apt-get install -y \
     zlib1g-dev
 ```
 
+With Ubuntu 22.04, cmake can be installed using `apt` as well:
```

Comment:
It's better to have pristine instructions, for 22 cmake can be included in the one-liner above.